GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -271,8 +276,33 @@ private void LoadFileHistory()
 
                     var lines = Module.GitExecutable.GetOutputLines(args, outputEncoding: GitModule.LosslessEncoding);
 
+                    ObjectId currentObjectId = null;
                     foreach (var line in lines.Select(GitModule.ReEncodeFileNameFromLossless))
                     {
+                        if (line.Length == 0)
+                        {
+                            // end of filenames
+                            currentObjectId = null;
+                            continue;
+                        }
+
+                        if (line.StartsWith(startOfObjectId))
+                        {
+                            currentObjectId = null;
+                            if (line.Length < ObjectId.Sha1CharCount + startOfObjectId.Length
+                                || !ObjectId.TryParse(line.Substring(startOfObjectId.Length, ObjectId.Sha1CharCount), 0, out currentObjectId))
+                            {
+                                // Parese error
+                                continue;
+                            }
+                        }
+
+                        if (!_revisionFilePath.ContainsKey(currentObjectId))
```

Comment:
This is .NET 5 now right? You can use TryAdd here instead.

Reply:
will change